application/Gemfile

```diff
@@ -15,18 +13,12 @@ gem "jbuilder"
 # Use Redis adapter to run Action Cable in production
 # gem "redis", ">= 4.0.1"
 
-# Use Faraday to access external APIs via HTTP
-gem "faraday"
-
```

Comment:
Not sure if there is still a dependency in the code that uses this gem. Did you ensure it is no longer used?

Reply:
good catch!